scripts/check.sh

```diff
@@ -163,29 +180,9 @@ if [[ "${VERBOSE}" == true ]]; then
   env | egrep '^TRAVIS_(BRANCH|COMMIT|PULL|REPO)' | sort || true
 fi
 
-# When travis clones a repo for building, it uses a shallow clone. After the
-# first commit on a non-master branch, TRAVIS_COMMIT_RANGE is not set and
-# START_REVISION is "origin/master" because a range wasn't passed in.
 if [[ "${START_REVISION}" == *..* ]]; then
-  RANGE_START="${START_REVISION/..*/}"
```

Comment:
I don't think this should be removed, or alternatively, this needs to be in the block where you handle `TRAVIS_COMMIT_RANGE`, if you specifically want to handle travis separately.

This was fixing a problem where travis would give us a commit range where the first commit was on some other branch and that's no longer being fixed in here.